packages/probing-response-updater/test/service.test.ts

```diff
@@ -0,0 +1,103 @@
+import { afterAll, beforeAll, describe, expect, it, vi } from "vitest";
+import { responseStatus } from "pagopa-interop-probing-models";
+import { createApiClient } from "../../probing-eservice-operations/src/model/generated/api.js";
+import {
+  EserviceService,
+  eServiceServiceBuilder,
+} from "../src/services/eserviceService.js";
+import {
+  AppError,
+  apiUpdateResponseReceivedError,
+  makeApplicationError,
+} from "../src/model/domain/errors.js";
+import { config } from "./../src/utilities/config.js";
+import { mockApiClientError } from "./utils.js";
+
+const apiClient = createApiClient(config.operationsBaseUrl);
+
+describe("eService service test", () => {
+  let eserviceService: EserviceService;
+
+  beforeAll(() => {
+    eserviceService = eServiceServiceBuilder(apiClient);
+  });
+
+  afterAll(() => {
+    vi.restoreAllMocks();
+  });
+
+  it("finding the e-service identified by eserviceRecordId, response received is successfully updated", async () => {
+    const eserviceRecordId = 1;
+    const status = responseStatus.ok;
+    const responseReceived = new Date(
+      Date.now() - new Date().getTimezoneOffset() * 60000
+    ).toISOString();
+
+    vi.spyOn(eserviceService, "updateResponseReceived").mockResolvedValue(
```

Comment:
I guess there is a typo, it should be
```suggestion
    vi.spyOn(apiClient, "updateResponseReceived").mockResolvedValue(
```
otherwise we would test the mock 😄 
(same for other tests)